### PR TITLE
SvaneMC

### PR DESCRIPTION
**Pull request by wiflow**
## Type of change

- [X] Add new directory for a server
- [x] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [X] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [X] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [X] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/svanemc/manifest.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "nice_name": "SvaneMC.net",
+  "direct_ip": "svanemc.net",
+  "supported_languages": [
+    "dk",
+    "en"
+  ],
+  "social": {
+    "web": "https://svanemc.net",
+    "web_shop": "https://buycraft.svanemc.net",
+    "web_support": "https://gommehdnet.zendesk.com",
```
Comment (DoJapHD) [inline minecraft_servers/svanemc/manifest.json:13]: I don't think this is yours

File: minecraft_servers/svanemc/manifest.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "discord": "https://discord.gg/svanemc"
+  },
+  "gamemodes": {
+    "Skyblock": {
```
Comment (DoJapHD) [inline minecraft_servers/svanemc/manifest.json:17]: JSON keys should be all lowercase

File: minecraft_servers/svanemc/manifest.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      "name": "Skyblock",
+      "command": "/server skyblock",
+      "versions": "1.21<*"
+    }
+  },
+  "discord": {
+    "server_id": 1278101104059613184,
+    "rename_to_minecraft_name": true
```
Comment (DoJapHD) [inline minecraft_servers/svanemc/manifest.json:25]: `true` is reserved for LabyMod Partners.
Please change to `false`

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/svanemc/manifest.json (hunk 3 above)
Comment (wiflow) [reply minecraft_servers/svanemc/manifest.json:24]: Hey Dojap,
Thank you for your feedback.

I've made the changes, but we are actually Labymod partners, - That's why it's still true.
Thanks again. :)

Comment (DoJapHD) [reply minecraft_servers/svanemc/manifest.json:24]: Sorry, my bad!

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**